src/ui/layout/index.md

```diff
@@ -188,6 +188,71 @@ class MyApp extends StatelessWidget {
   or you can build your own set of custom widgets.
 {{site.alert.end}}
 
+#### Cupertino apps
+
+To create a `Cupertino` app, use a [`CupertinoPageScaffold`][] widget.
+
+Unlike `Material`, it doesn't provide a default banner or background color.
+You need to set these yourself.
+
+* To set default colors, you add a [`CupertinoThemeData`][].
+* To add a [`CupertinoNavigationBar`][] as a `navigationBar` property value.
```

Comment:
I think the user's goal won't be to add a specific class/widget, but rather a certain UI element.

```suggestion
* To add an iOS-styled navigation bar to the top of your app, add a
  [`CupertinoNavigationBar`][] as the `navigationBar` of your scaffold.
```